react/wbooks/src/app/components/Login/styles.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '../../../scss/colors';
+
+@import '../../../scss/components';
```

Comment:
absolute import of scss doesn't work?

Reply:
By removing an import I get:
![image](https://user-images.githubusercontent.com/12807603/68916818-657cbc00-0746-11ea-9896-e2925277c50c.png)

Reply:
okk, np